src/o365/teams/commands/channel/teams-channel-remove.ts

```diff
@@ -0,0 +1,204 @@
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import Utils from '../../../../Utils';
+import request from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import { Channel } from '../../Channel';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  channelId?: string;
+  channelName?: string;
+  teamId: string;
+  confirm?: boolean;
+}
+
+class TeamsChannelRemoveCommand extends GraphCommand {
+  public get name(): string {
+    return commands.TEAMS_CHANNEL_REMOVE;
+  }
+
+  public get description(): string {
+    return 'Removes the specified channel in the Microsoft Teams team';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.channelId = args.options.channelId;
```

Comment:
`telemetryProps.channelId = args.options.channelId;` should chagne to something like `telemetryProps.channelId = typeof args.options.channelId !== 'undefined';`.

We track if the option is used only. We are not interested in the value itself since that could sometimes contain even PII.